sympy/printing/tests/test_latex.py

```diff
@@ -56,6 +56,8 @@ def test_latex_basic():
 
     assert latex(2*x*y) == "2 x y"
     assert latex(2*x*y, mul_symbol='dot') == r"2 \cdot x \cdot y"
+    assert latex(exp(-x)*log(x)) == r"e^{- x} \log{\left (x \right )}"
```

Comment:
This test was written to address the original issue. Since it now relates to tests for `fraction()`. I will remove them.